README.md

```diff
@@ -15,7 +15,7 @@ python setup.py install
 Add `datadog` to your list of installed apps.
 
 ```python
-INSTALLED_APPS += ('datadog')
+INSTALLED_APPS += ('django-datadog')
```

Comment:
The app should not have a dash (`-`) in its module name. Packages on PyPI typically use a _dash_ in their **package name** (e.g. [django-awesome-bootstrap](https://pypi.python.org/pypi/django-awesome-bootstrap)), and use an _underscore_ in their **module name** later (e.g. `awesome_bootstrap`), usually omitting the `django` prefix.

I believe this is supported by the Python packaging tool chain.